src/scripts/extensions/authenticationHelper.ts

```diff
@@ -136,12 +136,18 @@ export class AuthenticationHelper {
 	/**
 	 * Makes a call to the authentication proxy to retrieve the user's information.
 	 */
-	public static retrieveUserInformation(clipperId: string, cookie: string = undefined): Promise<ResponsePackage<string>> {
+	public static retrieveUserInformation(clipperId: string, cookie: string = undefined, logger: Logger = undefined): Promise<ResponsePackage<string>> {
```

Comment:
Agreed. I think we should try to cut down the use of static methods where we can.